sdk/digitaltwins/azure-digitaltwins-core/src/test/java/com/azure/digitaltwins/core/DigitalTwinsTestBase.java

```diff
@@ -120,4 +126,9 @@ protected DigitalTwinsAsyncClient getAsyncClient(HttpClient httpClient, DigitalT
             return Mono.just(new AccessToken("someFakeToken", OffsetDateTime.MAX));
         }
     }
+
+    // Used for converting json strings into BasicDigitalTwins, BasicRelationships, etc.
+    static <T> T deserializeJsonString(String rawJsonString, Class<T> clazz) throws JsonProcessingException {
+        return new ObjectMapper().readValue(rawJsonString, clazz);
+    }
```

Comment:
Our e2e tests that used the APIs that take objects rather than strings were still passing in strings. This didn't seem right to me, even if the tests still passed. I added this function so that when we use those APIs, we pass in objects like BasicDigitalTwin rather than the raw json string